include/mbedtls/ecdh.h

```diff
@@ -63,6 +93,15 @@ typedef struct mbedtls_ecdh_context
     mbedtls_ecp_point Vi;    /*!< The blinding value. */
     mbedtls_ecp_point Vf;    /*!< The unblinding value. */
     mbedtls_mpi _d;          /*!< The previous \p d. */
+#else
+    int point_format;        /*!< The format of point export in TLS messages. */
+    mbedtls_ecp_group_id grp;/*!< The elliptic curve used. */
+    mbedtls_ecdh_variant var;/*!< The ECDH implementation/structure used. */
```

Comment:
Please state explicitly that `var` indicates which field of `ctx` is used, either in the description of `var` or in the description of `ctx`.